app/src/main/java/com/example/lexis/fragments/FeedFragment.java

```diff
@@ -160,14 +185,17 @@ public void onSuccess(int statusCode, Headers headers, JSON json) {
                     String title = articleObject.getString("title");
                     String intro = articleObject.getString("extract");
                     intro = intro.replace("\n", "\n\n");
+                    String source = "Wikipedia";
```

Comment:
super tiny nit: Can you move this to above the articleObject.getString series? It makes the code more readable if similar functions are grouped together.

Reply:
```
String source = "Wikipedia";
String url = articleObject.getString("fullurl");
String title = articleObject.getString("title");
String intro = articleObject.getString("extract");
intro = intro.replace("\n", "\n\n");
```
etc